src/styles/_catalog.scss

```diff
@@ -1,9 +1,9 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
please specify the number of cards for all media queries. now it seems to be broken: 
![image](https://user-images.githubusercontent.com/22824431/235121411-ef06d1a5-0b04-4e69-9fb2-d6d3e16b4c1e.png)
![image](https://user-images.githubusercontent.com/22824431/235121576-bfca484c-4f57-4f44-9db4-2c9f8208eb2d.png)